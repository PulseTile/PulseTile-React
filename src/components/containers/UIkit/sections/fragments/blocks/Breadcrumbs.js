import React from 'react';

/**
 * This component returns content of Breadcrumbs block of UIkit page
 *
 * @return {XML}
 * @constructor
 */
const Breadcrumbs = () => {
  return (
    <div id="breadcrumbs" className="ui-section">
      <strong className="ui-title">Breadcrumbs</strong>
      <div className="ui-sub-section">
        <div className="wrap-breadcrumbs">
          <div className="container-fluid">
            <div className="breadcrumbs">
              <a className="breadcrumb-link">Patient Listings</a>
              <span className="breadcrumb-separate"></span>
              <a className="breadcrumb-link">Patient Summary</a>
              <span className="breadcrumb-separate"></span>
              <span className="breadcrumb-link active">Medications</span>
            </div>
          </div>
        </div>
       </div>
    </div>
  );
};

export default Breadcrumbs;
