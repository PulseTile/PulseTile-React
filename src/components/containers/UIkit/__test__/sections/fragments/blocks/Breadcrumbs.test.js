import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Breadcrumbs from '../../../../sections/fragments/blocks/Breadcrumbs';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Breadcrumbs />', () => {
    it('should renders Breadcrumbs with props correctly', () => {
        const component = shallow(<Breadcrumbs />);
        expect(component).toMatchSnapshot();
    });
});
