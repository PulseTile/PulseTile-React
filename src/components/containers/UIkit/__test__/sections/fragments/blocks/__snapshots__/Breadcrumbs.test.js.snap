// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Breadcrumbs /> should renders Breadcrumbs with props correctly 1`] = `
<div
  className="ui-section"
  id="breadcrumbs"
>
  <strong
    className="ui-title"
  >
    Breadcrumbs
  </strong>
  <div
    className="ui-sub-section"
  >
    <div
      className="wrap-breadcrumbs"
    >
      <div
        className="container-fluid"
      >
        <div
          className="breadcrumbs"
        >
          <a
            className="breadcrumb-link"
          >
            Patient Listings
          </a>
          <span
            className="breadcrumb-separate"
          />
          <a
            className="breadcrumb-link"
          >
            Patient Summary
          </a>
          <span
            className="breadcrumb-separate"
          />
          <span
            className="breadcrumb-link active"
          >
            Medications
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
